cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/token/RangeUtils.java

```diff
@@ -145,4 +160,26 @@ public static <Instance extends CassandraInstance> Multimap<Instance, Range<BigI
 
         return tokenRanges;
     }
+
+    @NotNull
+    public static TokenRange toTokenRange(@NotNull Range<BigInteger> range)
+    {
+        BigInteger lowerEndpoint = range.lowerEndpoint();
+        if (range.lowerBoundType() == BoundType.OPEN)
+        {
+            lowerEndpoint = lowerEndpoint.add(BigInteger.ONE);
+        }
+        BigInteger upperEndpoint = range.upperEndpoint();
+        if (range.upperBoundType() == BoundType.OPEN)
+        {
+            upperEndpoint = upperEndpoint.subtract(BigInteger.ONE);
+        }
+        return TokenRange.closed(lowerEndpoint, upperEndpoint);
+    }
+
+    @NotNull
+    public static Range<BigInteger> fromTokenRange(@NotNull TokenRange range)
```

Comment:
unused?

Reply:
unused and has been there. I will just keep it.